meganav.yaml

```diff
@@ -1,3 +1,699 @@
+            - title: PC and Laptops
+              description: Certified for Ubuntu
+              url: /certified?q=&limit=20&category=Laptop&category=Desktop
+            - title: Servers
+              description: Deploy on certified servers
+              url: /certified/servers
+            - title: IoT Devices
+              description: Stable and secure devices
+              url: /certified/devices
+            - title: System on a Chip
+              description: Supported in Ubuntu
+              url: /certified/socs
+            - title: Raspberry Pi devices
+              description: Certified Pi devices
+              url: /certified?q=&limit=20&vendor=Raspberry+Pi+Foundation
+
+      secondary_links:
+        - title: Links
+          links:
+            - title: Why use Certified devices?
+              url: /certified/why-certified
+            - title: Certify your device
+              url: https://canonical.com/partners/become-a-partner
+            - title: Why certification matters for AIoT
+              url: https://www.brighttalk.com/webcast/6793/497538
+            - title: Industrial Pi use cases webinar
+              url: https://www.brighttalk.com/webcast/6793/514263
+            - title: Why buy a preinstalled workstation?
+              url: /blog/why-you-should-buy-a-pre-installed-ubuntu-workstation
+
+    - title: IoT and Edge
+      primary_links:
+        - title: Products
+          links:
+            - title: Ubuntu Core
+              description: Tiny Embedded Ubuntu
+              url: /core
+            - title: Dedicated Snap Store
+              description: Private app store for Linux and IoT
+              url: http://snapcraft.io/
+            - title: Device management
+              url: /internet-of-things/management
+            - title: Certified devices
+              url: /certified/devices
+            - title: Ubuntu Frame
+              url: https://mir-server.io/ubuntu-frame
+            - title: IoT Professional Services
+              url: /core/smartstart
+
+      secondary_links:
+        - title: Links
+          links:
+            - title: Real-time Ubuntu
+              url: /blog/real-time-ubuntu-released
+            - title: Guide to IoT lifecycle management
+              url: /engage/a-practical-guide-to-iot-lifecycle-management
+            - title: Canonical joins the CSA
+              url: /blog/canonical-joins-the-connectivity-standards-alliance
+            - title: Pick the OS for your robots
+              url: /engage/robot-operating-system-choice
+            - title: "Embedded Linux: make or buy?"
+              url: /engage/embedded-linux-make-or-buy
+            - title: Try Ubuntu Core on Raspberry Pi
+              url: /download/raspberry-pi-core
+            - title: Industry 4.0
+              url: /engage/bridge-the-IT-OT-gap-Industry4
+
+use_case:
+  primary_links:
+    - title: By solution
+      links:
+        - title: AI/ML
```

Comment:
Are we not including descriptions for these?

Reply:
My bad - updating